frontend/app/views/fragments/joiner/signedInAs.scala.html

```diff
@@ -1,10 +1,12 @@
-@(returnUri: String)
+@(returnUri: String, inline: Boolean = false)
 
 @import configuration.Config.idWebAppSignOutThenInUrl
 
 <p class="text-note">
     You're signed in as <strong class="js-user-displayname"></strong>.
-</p>
-<p class="text-note">
-    <a href="@idWebAppSignOutThenInUrl(returnUri)" class="text-link">Sign out</a>
+@if(!inline) {
+	</p>
+	<p class="text-note">	
+}
+    <a href="@idWebAppSignOutThenInUrl(returnUri)" class="text-link">Sign&nbsp;out</a>
```

Comment:
@SiAdcock This &nbsp assures that in mobile view and if the username is really long, the 'sign out' will be always in the same line. Is there a better way to do this?